src/renderer/features/balances/subscription/model/balance-sub-model.ts

```diff
@@ -1,24 +1,25 @@
 import { ApiPromise } from '@polkadot/api';
 import { createEffect, createStore, sample, createEvent, scopeBind, attach } from 'effector';
-import { once, combineEvents, spread, previous } from 'patronum';
+import { once, combineEvents, spread, previous, interval } from 'patronum';
 import mapValues from 'lodash/mapValues';
 
 import { AccountId, Balance, ChainId, ConnectionStatus, Wallet, Chain, ID } from '@shared/core';
 import { accountUtils, walletModel } from '@entities/wallet';
 import { networkModel, networkUtils } from '@entities/network';
-import { balanceModel } from '@entities/balance';
+import { balanceModel, balanceUtils } from '@entities/balance';
 import { storageService } from '@shared/api/storage';
 import { balanceSubUtils } from '../lib/balance-sub-utils';
 import { Subscriptions, SubAccounts } from '../lib/types';
 import { balanceService } from '@shared/api/balances';
 import { isRejected } from '@shared/lib/utils';
 
-const balancesSubStarted = createEvent();
 const walletToUnsubSet = createEvent<Wallet>();
 const walletToSubSet = createEvent<Wallet>();
+const balancesUpdated = createEvent<Balance[]>();
 
 const $subscriptions = createStore<Subscriptions>({});
 const $subAccounts = createStore<SubAccounts>({});
+const $subBalances = createStore<Balance[]>([]);
```

Comment:
But I am not sure about name for this variable. Maybe add comment to better understand what is it about?